tests/ui/parser/attribute/attr-unquoted-ident.rs

```diff
@@ -1,17 +1,25 @@
 //@ compile-flags: -Zdeduplicate-diagnostics=yes
-//@ run-rustfix
```

Comment:
I've removed rustfix since I've added a negative test case and I didn't want to add revisions (do they work in combination with rustfix?) or a separate test file.